src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -19,8 +19,6 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public class SarifValidatorTests
     {
```

Comment:
It might be easier to read this file as new code, since it's so heavily refactored. The test for the "converter" directory and the "direct producer" directory were already very similar. The test for the new "spec samples" directory would have been identical to those for the direct producer directory. So aggregate the files for all three directories and write a single data-driven test.